tests/foreman/performance/test_standard_prep.py

```diff
@@ -32,7 +32,7 @@ def setUpClass(cls):
             'performance.test.manifest.location']
 
         # parameters for uploading manifests
-        cls.manifest_file = conf.properties['performance.test.manifest.file']
+        cls.manifest_file = conf.properties['main.manifest.fake_url']
```

Comment:
Please use get method here too. If you don't pass the second argument, like `conf.propertiesget('main.manifest.fake_url')`,  it will return None and then you can check if this var is None later on.